ipatests/create_external_ca.py

```diff
@@ -24,12 +24,19 @@
 from cryptography.hazmat.primitives.asymmetric import rsa
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import serialization
+import ipalib.constants
 
 import datetime
 import six
 
 ISSUER_CN = 'example.test'
 
+MAP_SIGNING_ALGORITHM = {
+    'SHA256withRSA': hashes.SHA256,
+    'SHA384withRSA': hashes.SHA384,
```

Comment:
I don't believe dogtag supports SHA384withRSA by default. I don't see it in defaultSigningAlgsAllowed in CS.cfg.

Reply:
Support for SHA384withRSA was added in Dogtag in June 2016 with https://github.com/dogtagpki/pki/commit/158bb22a87832ff2be07ac4b75c8f2927caefd55 so caCACert.cfg profile allows to use SHA384withRSA.

It is not yet allowed in CS.cfg by default, indeed. I need to check with the dogtag team why.

Reply:
I checked with the Dogtag team and it looks like this is a simple omission that will be fixed in dogtag. So we can keep SHA256withRSA for time being.
I verified that updating CS.cfg and defaulting to SHA384withRSA creates working IPA CA with 3072 bit key size and SHA384withRSA signing algorithm.